src/FondOfSpryker/Yves/EnhancedEcommerceCheckoutConnector/EnhancedEcommerceCheckoutConnectorDependencyProvider.php

```diff
@@ -2,17 +2,56 @@
 
 namespace FondOfSpryker\Yves\EnhancedEcommerceCheckoutConnector;
 
+use FondOfSpryker\Yves\EnhancedEcommerceCheckoutConnector\Dependency\EnhancedEcommerceCheckoutConnectorToCartClientBridge;
 use Spryker\Yves\Kernel\AbstractBundleDependencyProvider;
 use Spryker\Yves\Kernel\Container;
+use Spryker\Yves\Money\Plugin\MoneyPlugin;
 
 class EnhancedEcommerceCheckoutConnectorDependencyProvider extends AbstractBundleDependencyProvider
 {
+    public const CART_CLIENT = 'CART_CLIENT';
+    public const PLUGIN_MONEY = 'PLUGIN_MONEY';
+
     /**
-     * @param Container $container
-     * @return Container
+     * @param \Spryker\Yves\Kernel\Container $container
+     *
+     * @return \Spryker\Yves\Kernel\Container
      */
     public function provideDependencies(Container $container): Container
     {
-        return parent::provideDependencies($container); // TODO: Change the autogenerated stub
+        $container = $this->addCartClient($container);
+        $container = $this->addMoneyPlugin($container);
+
+        return $container;
+    }
+
+    /**
+     * @param \Spryker\Yves\Kernel\Container $container
+     *
+     * @return \Spryker\Yves\Kernel\Container
+     */
+    protected function addCartClient(Container $container): Container
+    {
+        $container->set(static::CART_CLIENT, static function (Container $container) {
+            return new EnhancedEcommerceCheckoutConnectorToCartClientBridge(
+                $container->getLocator()->cart()->client()
+            );
+        });
+
+        return $container;
+    }
+
+    /**
+     * @param \Spryker\Yves\Kernel\Container $container
+     *
+     * @return \Spryker\Yves\Kernel\Container
+     */
+    protected function addMoneyPlugin(Container $container): Container
+    {
+        $container->set(static::PLUGIN_MONEY, static function () {
+            return new MoneyPlugin();
+        });
+
+        return $container;
```

Comment:
Does money maybe have a client as well ?

Reply:
yes, there is a money-client. but i dosent provide a simple function to format integer to float. there are always some other properties needed. if i use the client i need additional dependencies too, i am not sure if this is better as using the plugin.

Reply:
Probably `MoneyFacadeInterface` might be right then.. 

```/**
     * Specification
     * - Converts an integer value into decimal value
     *
     * @api
     *
     * @param int $value
     *
     * @return float
     */
    public function convertIntegerToDecimal($value);```